exonum/src/node/basic.rs

```diff
@@ -220,6 +220,7 @@ impl NodeHandler {
     /// Node update internal `ApiState`.
     pub fn handle_update_api_state_timeout(&mut self) {
         self.api_state.update_node_state(&self.state);
+        self.api_state.update_majority_count(&self.state);
```

Comment:
Questionable: maybe make this method private and call it from the `update_node_state`? As far as I can see, those are should be called together.

Reply:
Yes, it make sense